### PR TITLE
replace ping_check and mtu_set with configure_network module

### PR DESCRIPTION
**Pull request by smruti77**
replace ping_check, mtu_set in network_test file and add mu_set to
iperf_test, uperf_test, netperf_test file.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

**Comment by smruti77** [comment]
JOB ID     : d9f5733da0b59891d2568e65a62eac9bc013d885
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-10-11T16.22-d9f5733/job.log
 (01/18) network_test.py:NetworkTest.test_gro;1500-02f3: PASS (10.73 s)
 (02/18) network_test.py:NetworkTest.test_gro;2000-316b: PASS (10.67 s)
 (03/18) network_test.py:NetworkTest.test_lro;1500-02f3: FAIL: Can not change the state of lro (2.26 s)
 (04/18) network_test.py:NetworkTest.test_lro;2000-316b: FAIL: Can not change the state of lro (2.29 s)
 (05/18) network_test.py:NetworkTest.test_ping;1500-02f3: PASS (6.41 s)
 (06/18) network_test.py:NetworkTest.test_ping;2000-316b: PASS (6.36 s)
 (07/18) network_test.py:NetworkTest.test_floodping;1500-02f3: PASS (2.24 s)
 (08/18) network_test.py:NetworkTest.test_floodping;2000-316b: PASS (2.29 s)
 (09/18) network_test.py:NetworkTest.test_ssh;1500-02f3: PASS (2.40 s)
 (10/18) network_test.py:NetworkTest.test_ssh;2000-316b: PASS (2.44 s)
 (11/18) network_test.py:NetworkTest.test_scp;1500-02f3: PASS (32.76 s)
 (12/18) network_test.py:NetworkTest.test_scp;2000-316b: PASS (26.00 s)
 (13/18) network_test.py:NetworkTest.test_jumbo_frame;1500-02f3: PASS (5.42 s)
 (14/18) network_test.py:NetworkTest.test_jumbo_frame;2000-316b: PASS (5.46 s)
 (15/18) network_test.py:NetworkTest.test_statistics;1500-02f3: PASS (6.35 s)
 (16/18) network_test.py:NetworkTest.test_statistics;2000-316b: PASS (6.44 s)
 (17/18) network_test.py:NetworkTest.test_promisc;1500-02f3: PASS (10.67 s)
 (18/18) network_test.py:NetworkTest.test_promisc;2000-316b: PASS (10.67 s)
RESULTS    : PASS 16 | ERROR 0 | FAIL 2 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 152.69 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-10-11T16.22-d9f5733/results.html
JOB ID     : 870a23f89616162642ecf0420cc0f52d57edf3fd
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-10-11T16.27-870a23f/job.log
 (1/1) iperf_test.py:Iperf.test;run-6eed: FAIL: FAIL: Iperf Run failed (22.32 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 22.41 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-10-11T16.27-870a23f/results.html
JOB ID     : 05158b982f5e34d9de6eab7f4d5d2bd5c54aeac2
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-10-11T16.27-05158b9/job.log
 (1/1) uperf_test.py:Uperf.test;run-ed29: FAIL: FAIL: Uperf Run failed (13.15 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 13.25 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-10-11T16.27-05158b9/results.html

JOB ID     : 4c802ae631db95f9f67c8ca77cc1bbb5a8ab82c6
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-10-11T15.22-4c802ae/job.log
 (1/5) netperf_test.py:Netperf.test;generic-1729: PASS (499.49 s)
 (2/5) netperf_test.py:Netperf.test;tcp_stream-b6d4: PASS (32.76 s)
 (3/5) netperf_test.py:Netperf.test;udp_stream-89f2: FAIL: FAIL: Throughput Actual - 40.06%, Expected - 90%, Throughput Actual value - 4006Mb/sec  (26.15 s)
 (4/5) netperf_test.py:Netperf.test;tcp_rr-0ce4: WARN: Test passed but there were warnings during execution. Check the log for details. (616.08 s)
 (5/5) netperf_test.py:Netperf.test;udp_rr-6a6a: WARN: Test passed but there were warnings during execution. Check the log for details. (616.11 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 2 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1790.89 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-10-11T15.22-4c802ae/results.html

**Comment by narasimhan-v** [comment]
@smruti77 any updates on the avocado utils ?

**Comment by smruti77** [comment]
> @smruti77 any updates on the avocado utils ?

yesterday that was merged.

**Comment by narasimhan-v** [comment]
https://github.com/avocado-framework/avocado/pull/3335

**Comment by narasimhan-v** [comment]
@smruti77 Can you please paste job.log with latest avocado code ?
Once done, it is ready for merge.

**Comment by smruti77** [comment]
JOB ID     : 1f7a5312847b4cfe7a4cef5865209df8ed2e0d8a
JOB LOG    : /root/avocado/job-results/job-2019-11-04T10.47-1f7a531/job.log
 (1/9) network_test.py:NetworkTest.test_gro;1500-3229: PASS (10.80 s)
 (2/9) network_test.py:NetworkTest.test_lro;1500-3229: FAIL: Can not change the state of lro (2.20 s)
 (3/9) network_test.py:NetworkTest.test_ping;1500-3229: PASS (6.42 s)
 (4/9) network_test.py:NetworkTest.test_floodping;1500-3229: PASS (2.25 s)
 (5/9) network_test.py:NetworkTest.test_ssh;1500-3229: PASS (2.36 s)
 (6/9) network_test.py:NetworkTest.test_scp;1500-3229: PASS (26.46 s)
 (7/9) network_test.py:NetworkTest.test_jumbo_frame;1500-3229: PASS (5.42 s)
 (8/9) network_test.py:NetworkTest.test_statistics;1500-3229: PASS (6.39 s)
 (9/9) network_test.py:NetworkTest.test_promisc;1500-3229: PASS (10.69 s)
RESULTS    : PASS 8 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 73.49 s
JOB HTML   : /root/avocado/job-results/job-2019-11-04T10.47-1f7a531/results.html

JOB ID     : bde9467f5663337f7bef9c6647759a585a2ed567
JOB LOG    : /root/avocado/job-results/job-2019-11-04T11.11-bde9467/job.log
 (1/1) iperf_test.py:Iperf.test;run-2a98: FAIL: FAIL: Throughput Actual - 0.009%, Expected - 90%, Throughput Actual value - 9Mb/sec  (43.21 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 43.31 s
JOB HTML   : /root/avocado/job-results/job-2019-11-04T11.11-bde9467/results.html

JOB ID     : 1568c878b7c5630facd720bc37a3d641d83d7c41
JOB LOG    : /root/avocado/job-results/job-2019-11-04T11.08-1568c87/job.log
 (1/1) uperf_test.py:Uperf.test;run-591c: FAIL: FAIL: Throughput Actual - 8.0%, Expected - 80%, Throughput Actual value - 8000Mb/sec  (54.02 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 54.12 s
JOB HTML   : /root/avocado/job-results/job-2019-11-04T11.08-1568c87/results.html

JOB ID     : d092094d90345c74ed3b50776af7b7e11d4b266a
JOB LOG    : /root/avocado/job-results/job-2019-11-04T10.56-d092094/job.log
 (1/5) netperf_test.py:Netperf.test;generic-e747: FAIL: FAIL: Throughput Actual - 9.414%, Expected - 90%, Throughput Actual value - 9414Mb/sec  (389.32 s)
 (2/5) netperf_test.py:Netperf.test;tcp_stream-898a: FAIL: FAIL: Throughput Actual - 7.01%, Expected - 90%, Throughput Actual value - 7010Mb/sec  (38.56 s)
 (3/5) netperf_test.py:Netperf.test;udp_stream-1cc2: FAIL: FAIL: Throughput Actual - 4.209%, Expected - 90%, Throughput Actual value - 4209Mb/sec  (37.19 s)
 (4/5) netperf_test.py:Netperf.test;tcp_rr-681f: WARN: Test passed but there were warnings during execution. Check the log for details. (627.13 s)
 (5/5) netperf_test.py:Netperf.test;udp_rr-0e1e: WARN: Test passed but there were warnings during execution. Check the log for details. (627.17 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 3 | SKIP 0 | WARN 2 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1719.71 s
JOB HTML   : /root/avocado/job-results/job-2019-11-04T10.56-d092094/results.html

**Comment by PraveenPenguin** [comment]
@smruti77 please squash all three commits in single commit may you need to rebase

File: io/net/uperf_test.py.data/uperf_test.yaml [hunk 1]
```diff
@@ -6,3 +6,4 @@ peer_user_name: "root"
 peer_password: "********"
 EXPECTED_THROUGHPUT : 80
 UPERF_SERVER_RUN : 1
+mtu : "1500"
```
Comment (narasimhan-v) [inline io/net/uperf_test.py.data/uperf_test.yaml]: @smruti77 Also, run this for all mtu sizes

File: io/net/netperf_test.py.data/netperf_test.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ EXPECTED_THROUGHPUT: 90
 duration: 120
 minimum_iterations: 1
 maximum_iterations: 5
+mtu: "1500"
```
Comment (narasimhan-v) [inline io/net/netperf_test.py.data/netperf_test.yaml]: @smruti77 Also, run this for all mtu sizes

Comment (smruti77) [reply io/net/netperf_test.py.data/netperf_test.yaml]: and for iperf ..?

Comment (narasimhan-v) [reply io/net/netperf_test.py.data/netperf_test.yaml]: yes please

**Comment by smruti77** [comment]
> @smruti77 Code looks fine. But the commits still need to be squashed into 1.

how to do that

**Pull request closed without merging**